packages/typedoc-plugin-appium/lib/logger.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Adapted from `@knodes/typedoc-pluginutils`
+ *
+ * Portions Copyright (c) 2022 KnodesCommunity
+ * Licensed MIT
+ *
+ * @module
+ * @see https://github.com/KnodesCommunity/typedoc-plugins/blob/ed5e4e87f5d80abf6352e8de353ea376c4f7db6d/packages/pluginutils/src/plugin-logger.ts
+ *
+ */
+
+import {format} from 'node:util';
+import {Logger, LogLevel} from 'typedoc';
+
+const LogMethods = {
+  [LogLevel.Error]: 'error',
+  [LogLevel.Warn]: 'warn',
+  [LogLevel.Info]: 'info',
+  [LogLevel.Verbose]: 'verbose',
+} as const;
+
+export class AppiumPluginLogger extends Logger {
```

Comment:
is this logger specific for this plugin or it is supposed to be used by other plugins as well?

Reply:
it's specific to this plugin. why?

Reply:
I thought we could move it to base plugin. Nvm, if it is specific

Reply:
oh, yeah, no, it's not an appium logger. it's typedoc-specific.